addons/main/functions/fn_createTable.sqf

```diff
@@ -39,6 +39,10 @@ params [
     ["_manualZoffset", 0]
 ];
 
+	//Loading screen environment has more computing power
+	startLoadingScreen ["Loading 3D table.."]; //text is actually unused
+	uiSleep 0.1; //Give engine some space to render the loading screen rsc
```

Comment:
Not guaranteed to be called in scheduled environment.
` 
if (canSuspend) then {uiSleep 0.1};?`